### ISSUE TITLE
New state files not installed during update

### ISSUE THREAD

**Issue by phmccarty** [clearlinux/swupd-client#535]
Today, my system successfully updated from 23940 to 24120. But after the update, I ran `verify --fix`, and it installed 14 missing files:

```
/usr/src/sysdig-0.1.1dev/bpf
/usr/src/sysdig-0.1.1dev/bpf/Makefile
/usr/src/sysdig-0.1.1dev/bpf/bpf_helpers.h
/usr/src/sysdig-0.1.1dev/bpf/filler_helpers.h
/usr/src/sysdig-0.1.1dev/bpf/fillers.h
/usr/src/sysdig-0.1.1dev/bpf/maps.h
/usr/src/sysdig-0.1.1dev/bpf/plumbing_helpers.h
/usr/src/sysdig-0.1.1dev/bpf/probe.c
/usr/src/sysdig-0.1.1dev/bpf/quirks.h
/usr/src/sysdig-0.1.1dev/bpf/ring_helpers.h
/usr/src/sysdig-0.1.1dev/bpf/types.h
/usr/src/sysdig-0.1.1dev/fillers_table.c
/usr/src/sysdig-0.1.1dev/ppm_fillers.h
/usr/src/sysdig-0.1.1dev/ppm_flag_helpers.h
```

My system was clean before updating, since a ran `verify --fix` beforehand.

Digging into the manifests a bit, I see all of these files are "state" files (i.e. the third field in the first column is `s`). Maybe the files are being skipped due to the current heuristics in place?

**Comment by jwakre** [comment]
Yes, the current heuristics skip over these files because they exist in the /usr/src directory. @matthewrsj @otaviobp What is the expected behavior for state files included in manifests? Should these be pulled into the system by bundle-add and update operations (they are not now)? In which cases should they be ignored?

**Comment by otaviobp** [comment]
According to the best definition I found about state files this behavior is correct.

The old documentation on swupd-server states that "State files are kept in manifests, but they are always ignored by swupd-client." [1]

But instead of closing this bug I'll change the label to an enhancement, because we need to review if this documentation makes sense and update the swupd documentation to state that behavior. I also created an umbrella issue to group state/boot file problems (#789).

1: https://github.com/clearlinux/swupd-server/wiki/File-heuristics
